.golangci.yml

```diff
@@ -110,6 +110,7 @@ linters:
           - recvcheck
         path: managed/models/
     paths:
+      - managed/utils/interceptors/go-grpc-prometheus.go
```

Comment:
Do we to leave this disabled or is it a temp workaround?

Reply:
I would leave it disabled (a lot of lint issues there) until go-grpc-prometheus is replaced by go-grpc-middleware. Code in this file will be deleted/rewritten.